### DISCUSSION TITLE
Does Web3Auth support an incognito mode?

### DISCUSSION THREAD

**Discussion by chaitanyapotti** [Web3Auth/Web3Auth#334] [Feature Requests]
Considering that people link their ens names etc to blockchain accounts, does web3auth support an incognito mode (similar to apple privacy mode) where we can get an anonymised key for an app (different from regular key) and not share any social data?

**Comment by yashovardhan** [comment]
In Web3Auth, if your [`clientId`](https://web3auth.io/docs/developer-dashboard/get-client-id) and/ or [`verifier`](https://web3auth.io/docs/custom-authentication/verifiers) is different, you will get a different private key. This means for every different dApp and social login combination, you will get a different private key.

Hence you can use your apple private mode to even login into web3auth and customise what information you want to share in the first place. You can even create new email account to do that. Passwordless login can also work here. The dApp will still have access to the `getUserInfo` function that gives them the information about the user, but the information shared with Web3Auth in the first place will be extremely limited and hence it can work as an incognito login.
